internal/services/fluidrelay/sdk/2022-05-26/fluidrelayservers/method_getkeys_autorest.go

```diff
@@ -0,0 +1,65 @@
+package fluidrelayservers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+)
+
+type GetKeysOperationResponse struct {
+	HttpResponse *http.Response
+	Model        *FluidRelayServerKeys
+}
+
+// GetKeys ...
+func (c FluidRelayServersClient) GetKeys(ctx context.Context, id FluidRelayServerId) (result GetKeysOperationResponse, err error) {
```

Comment:
GetKeys is deprecated. Please remove this. It is actually removed in the 2022-06-01 version which we prefer to use if possible like I mentioned in the ADO item.

Reply:
thanks for your reminder, this code was generated by tools, and we do not call it